### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by noahmarxx**

### CODE REVIEW

File: backend/backend.js [hunk 1]
```diff
@@ -46,6 +45,12 @@ app.get("/pins", async (req, res) => {
   }
 });
 
+app.get("/pins/:id", async (req, res) => {
```
Comment (davidayu) [inline backend/backend.js:48]: I believe these changes are already merged to main. Can you check to see why these additions are here?

Comment (noahmarxx) [reply backend/backend.js:48]: Just merged with main and fixed the problem. Thanks David.

File: backend/models/comment-services.js [hunk 2]
```diff
@@ -1,19 +1,53 @@
 const mongoose = require("mongoose");
-const commentModel = require("./comment");
+const test = 0;
+const commentSchema = require("./comment");
 const pinModel = require("./pin");
 const dotenv = require("dotenv").config({ path: "database.env" });
 
+let conn;
```
Comment (eliasow) [inline backend/models/comment-services.js:7]: this variable name could be more clear

**Review by davidayu — APPROVED** [review]

File: backend/models/comment-services.js (hunk 2 above)
Comment (noahmarxx) [reply backend/models/comment-services.js:7]: Thanks for the tip!

**Comment by AmeyaMalekar** [comment]
Nice work, looks good.

**Comment by ksittner** [comment]
Looked good to me, thank you for unit testing

**Comment by noahmarxx** [comment]
> Nice work, looks good.

Thanks for the feedback Ameya

**Comment by noahmarxx** [comment]
> Looked good to me, thank you for unit testing

You're welcome Kevin!

**Pull request merged**